### ISSUE TITLE
Unable to pass CapabilityPath or StoragePath as arg to Cadence Transaction

### ISSUE THREAD

**Issue by jacksonConrad** [onflow/flow-js-testing#79]
### Problem

It seems that passing in StoragePath, PrivatePath, or PublicPath as an argument to a Cadence script or transaction does not work.  I get the following error:

```
TypeError: Cannot destructure property 'status' of 'n[0]' as it is null.
```
Additionally, when running the emulator in my testing code w/ the verbose flag `true`, there is no emulator output.  So I believe the bug is within `flow-js-testing` itself.

### Steps to Reproduce

I have the following transaction signature:

`transaction(adminRequesterAddress: Address, storPath: StoragePath, privPath: PrivatePath)`

I'm passing in the following args to sendTransaction: `[ '0x179b6b1cb6755e31', '/storage/things', '/private/things' ]`
(transaction body doesnt really matter, it can be empty and just return an int)

Code in my test file:
```
const signers = [DeployerAccount]
const args = [acct, `/storage/things`, `/private/things`]
const txResult = await shallPass(
    sendTransaction({ code: DistributeAdminCapabilityCode, args, signers })
)
```

When running the test suite, I get the following error:

```
TypeError: Cannot destructure property 'status' of 'n[0]' as it is null.
```

**Comment by MaxStalker** [comment]
@jacksonConrad can you provide portion of Cadence code as well - specifically parameters it expects :)

**Comment by jacksonConrad** [comment]
Sure thing!! I included the transaction signature above, but here is the whole cadence transaction for posterity :)

`
import VotingCommunity from "../contracts/VotingCommunity.cdc"

transaction(adminRequesterAddress: Address, storPath: StoragePath, privPath: PrivatePath) {

    let resourceStoragePath: StoragePath
    let capabilityPrivatePath: CapabilityPath
    let adminCapability: Capability<&VotingCommunity.CommunityCollection>
    let adminRequester: PublicAccount

    prepare(superAdmin: AuthAccount) {

        self.adminRequester = getAccount(adminRequesterAddress)

        // These paths must be unique within the contract account's storage
        // TODO: use programatic Paths when supported, or construct on client and pass in
        // self.resourceStoragePath = StoragePath(identifier: adminRequesterAddress.toString())
        // self.capabilityPrivatePath = PrivatePath(identifier: adminRequesterAddress.toString())
        // self.resourceStoragePath = /storage/admin_01
        // self.capabilityPrivatePath = /private/admin_01
        self.resourceStoragePath = storPath
        self.capabilityPrivatePath= privPath

        let superAdminCap = superAdmin.borrow<&VotingCommunity.SuperAdmin>(from: VotingCommunity.SUPER_ADMIN_PATH) ??
            panic("Could not find Super Admin resource")

        // create collection to be linked
        let collection <- superAdminCap.createCollection()
        superAdmin.save(<- collection, to: self.resourceStoragePath)

        // link capability
        self.adminCapability = superAdmin.link<&VotingCommunity.CommunityCollection>(self.capabilityPrivatePath, target: self.resourceStoragePath) ??
            panic("Could not link admin capability")
        
        // track the capability in-contract
        superAdminCap.trackCapability(adminAddress: self.adminRequester.address, storagePath: self.resourceStoragePath)

    }

    execute {

        let capabilityReceiver = self.adminRequester.getCapability
            <&VotingCommunity.AdminProxy{VotingCommunity.AdminReceiver}>
            (VotingCommunity.ADMIN_PROXY_PUBLIC_PATH)
            .borrow() ?? panic("Could not borrow capability receiver reference")

        capabilityReceiver.setCapability(capability: self.adminCapability)
    }

}
`

**Comment by jribbink** [comment]
This should be fixed in @onflow/flow-js-testing@0.3.0-alpha.10
